test/k8sT/Services.go

```diff
@@ -2056,6 +2059,15 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 			})
 		})
 
+		// SA needs 4.19. However, on the 4.19 job we run KPR. So to test SA on
+		// bpf_lxc with kube-proxy we need to disable KRP.
+		SkipItIf(helpers.DoesNotRunOn419Kernel, "Tests kube-proxy ClusterIP with sessionAffinity from pods", func() {
```

Comment:
```suggestion
		SkipItIf(func() bool {
			return helpers.RunsWithKubeProxy() && helpers.RunsOn419OrLaterKernel()
		}, "Tests kube-proxy ClusterIP with sessionAffinity from pods", func() {
```
And I think you can then make the comment shorter if you want :-)